test/schema_test.exs

```diff
@@ -104,6 +106,24 @@ defmodule OpenApiSpex.SchemaTest do
       assert {:error, _} = Schema.cast(user_request_schema, input, schemas)
     end
 
+    test "cast request schema error with custom validator" do
+      api_spec = ApiSpec.spec()
```

Comment:
Instead of adding `add: StrictInt.schema()` to the `User` schema `test/support/schemas.ex`, I think it would be simpler to define a schema inline in this test:

```elixir
    test "cast request schema error with custom validator" do
      user_request_schema = %Schema{
        type: :object,
        properties: %{age: OpenApiSpexTest.Schemas.StrictInt.schema()}
      }

      input = %{"age" => "83"}

      assert {:error, _} = Schema.cast(user_request_schema, input, %{})
  end
```

It's difficult to trace back to the definition of `User` in `test/support/schemas.ex`, and that schema is depended on by too many tests. Also, this test is dealing with more data and schemas than it needs to be.

Once you've done that, you can revert the little changes that were needed when the `age` property was added.